geomstats/special_euclidean_group.py

```diff
@@ -259,35 +286,43 @@ def jacobian_translation(
         dim_rotations = rotations.dimension
 
         point = self.regularize(point, point_type=point_type)
-        n_points, _ = point.shape
 
-        rot_vec = point[:, :dim_rotations]
+        if point_type == 'vector':
+            n_points, _ = point.shape
+
+            rot_vec = point[:, :dim_rotations]
 
-        jacobian = gs.zeros((n_points,) + (dim,) * 2)
+            jacobian = gs.zeros((n_points,) + (dim,) * 2)
 
-        if left_or_right == 'left':
-            jacobian_rot = self.rotations.jacobian_translation(
+            if left_or_right == 'left':
+                jacobian_rot = self.rotations.jacobian_translation(
```

Comment:
you should be able to remove this line from the if statement and just pass left_or_right=left_or_right